assets/airports/kpvd.json

```diff
@@ -0,0 +1,6536 @@
+        "STUFY": ["N41d40m37.2000", "W071d38m01.1900"],
+        "STURB": ["N42d02m16.5700", "W071d57m17.3100"],
+        "SUFOK": ["N41d09m47.6100", "W071d49m44.1000"],
+        "TAMAE": ["N41d47m57.7200", "W072d48m12.0600"],
+        "TENEE": ["N41d33m42.2400", "W072d46m41.3400"],
+        "THUMB": ["N41d31m23.9600", "W072d21m31.8000"],
+        "TRAIT": ["N41d17m04.7500", "W071d55m03.3500"],
+        "UJEFO": ["N41d30m22.9000", "W071d34m06.4300"],
+        "UNTER": ["N41d50m17.0600", "W071d27m53.1000"],
+        "UTUBE": ["N41d36m34.8300", "W072d52m28.7600"],
+        "UZAZU": ["N41d54m18.4500", "W072d04m31.8900"],
+        "VPBIG": ["N41d05m40.0000", "W071d55m00.0000"],
+        "VPMLT": ["N41d04m40.0000", "W071d51m00.0000"],
+        "VPRES": ["N41d41m45.8300", "W072d21m04.5700"],
+        "WACKY": ["N41d20m25.5200", "W071d51m22.3200"],
+        "WALLU": ["N41d44m35.9300", "W071d38m26.7100"],
+        "WANDI": ["N41d22m33.7700", "W072d08m01.8300"],
+        "WANTU": ["N42d20m54.9200", "W071d03m11.8500"],
+        "WAPOG": ["N41d25m30.5800", "W071d54m43.3300"],
+        "WAVON": ["N41d11m12.6900", "W071d20m48.3800"],
+        "WAWIK": ["N41d37m11.5800", "W071d39m09.4500"],
+        "WEGOT": ["N41d32m25.3800", "W072d35m32.1900"],
+        "WERID": ["N41d13m06.5900", "W070d46m38.6400"],
+        "WHISL": ["N41d50m10.6000", "W072d47m18.5500"],
+        "WIKIT": ["N41d38m53.4200", "W071d29m23.9400"],
+        "WILIM": ["N41d36m59.1000", "W072d47m22.7800"],
+        "WIPOR": ["N41d39m23.0700", "W072d10m37.6700"],
+        "WIVPA": ["N41d45m10.9900", "W071d24m07.2500"],
+        "WOMUL": ["N41d33m17.2400", "W072d55m50.0100"],
+        "WOONS": ["N41d57m02.0700", "W071d30m25.3400"],
+        "XOBIJ": ["N41d54m08.2200", "W072d48m48.7500"],
+        "YBERA": ["N41d38m20.6100", "W071d31m29.9800"],
+        "YELOW": ["N41d55m49.0700", "W072d51m17.7000"],
+        "YODEL": ["N41d23m25.3700", "W072d39m30.5200"],
+        "YODER": ["N41d17m22.5100", "W072d20m32.7900"],
+        "ZADOR": ["N41d39m56.9700", "W072d48m00.3200"],
+        "ZAGAR": ["N42d00m21.2300", "W072d03m24.0800"],
+        "ZARAS": ["N41d43m56.6700", "W072d14m16.3100"],
+        "ZEPVI": ["N41d34m33.7300", "W072d52m43.9100"],
+        "ZOVOX": ["N41d42m13.0600", "W072d36m52.9500"],
+        "_MAN" : ["N41d42m13.0600", "W072d36m52.9500"],
+        "_ONE" : ["N00d00m00.0000", "W000d00m00.0000"]
+    },
+    "restricted": [
+        {
+            "name": "R4 102A",
+            "height": "2000ft",
+            "coordinates": [
+                ["N42d31m15", "W071d36m28"], 
+                ["N42d31m00", "W071d39m13"], 
+                ["N42d30m45", "W071d39m13"], 
+                ["N42d30m15", "W071d39m58"], 
+                ["N42d29m45", "W071d41m13"], 
+                ["N42d28m15", "W071d41m13"], 
+                ["N42d28m00", "W071d39m43"], 
+                ["N42d29m45", "W071d37m43"]
+            ]
+        },
+        {
+            "name": "R4 102B",
```

Comment:
These are the exact same other than the ceiling so I'd just merge them into one with a ceiling of 3995ft (like in BOS)